pages/resources/voice.mdx

```diff
@@ -148,3 +148,33 @@ Sending a voice channel effect requires the current user to be connected to the
 | ----- | ------- | -------------------------------------------------------- |
 | 0     | PREMIUM | A fun animation, requires a premium (Nitro) subscription |
 | 1     | BASIC   | The standard animation                                   |
+
+<RouteHeader method="GET" url="/streams/{stream_key}/preview">
+  Get Stream Preview
+</RouteHeader>
+
+Returns a URL to a stream preview for the given [stream key](/topics/gateway-events#stream-key).
```

Comment:
I'm pretty sure this endpoint doesn't work if the current user is not in the same guild / gdm/dm as the stream, maybe this should be noted?